README.md

```diff
@@ -80,7 +80,7 @@ Make sure you have read and write permissions on the ```/dev/usb/*``` device tha
 ## Install
 
 ```sh
-npm install bleno
+npm install bleno@npm:@abandonware/bleno
```

Comment:
good maybe we have @abandonware prefix elsewhere because so far we used

    npm install @abandonware/bleno

Please also double check in related projects